cluster/common.sh

```diff
@@ -908,7 +911,6 @@ EOF
      [[ "${master}" == "false" && "${NODE_OS_DISTRIBUTION}" == "container-linux" ]]; then
     # Container-Linux-only env vars. TODO(yifan): Make them available on other distros.
     cat >>$file <<EOF
-KUBERNETES_CONTAINER_RUNTIME: $(yaml-quote ${CONTAINER_RUNTIME:-rkt})
```

Comment:
Should we keep this for backward compatibility for at least a release?

Reply:
OK. Will keep the container-linux side unchanged.

Reply:
I mean to accept both for now :-)